config/environments/production.rb

```diff
@@ -100,26 +100,11 @@
     port: 25
   }
 
-  config.middleware.use ExceptionNotification::Rack,
-  :email => {
-    :email_prefix => "[ERROR] ",
-    :sender_address => %{"notifier" <parking@umass.edu>},
-    :exception_recipients => %w{parking-it@umass.edu}
-  },
-  :slack => {
-    :webhook_url => "https://hooks.slack.com/services/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",
-    :channel => "#probably-exceptions",
-    :additional_parameters => {
-      :icon_url => "http://image.jpg",
-      :mrkdwn => true
-    }
-  }
-
   # Inventory api url
   config.inventory_api_uri = 'https://rentalapi.parking.umass.edu/v1/' # not sure what this is yet
 end
 
-GolfCartRentals::Application.config.middleware.user ExceptionNotification::Rack,
+GolfCartRentals::Application.config.middleware.use ExceptionNotification::Rack,
```

Comment:
I think you can just use the `config` declared above, as in all our other apps. No need to re-declare the `GolfCartRentals::Application` here.